src/scala/microsoft-spark-3-5/pom.xml

```diff
@@ -0,0 +1,91 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```

Comment:
Is this file copied from src/scala/microsoft-spark-3-2/pom.xml? Can you try to move it and then modify? Similar to this: https://github.com/dotnet/spark/commit/80c745bd4099ee4fe28899c85b14ffa2b26e4756

Reply:
It highlights what is changed.

Reply:
Sure, it's already done, https://github.com/dotnet/spark/pull/1178/commits/e7eccdfdbc5be41ef66e8507107aba2a3153b2b4

Reply:
Looks good.